registrering-gui/src/app/dataset/dataset.ts

```diff
@@ -55,5 +55,7 @@ export interface Dataset {
 
   temporals?: PeriodOfTime[];
 
+  type?: string;
```

Comment:
lurer generelt på om det riktig av oss å bruke norsk streng for typene, men det får bli noe vi diskuterer senere